doc/usage/Getting_Started.md

```diff
@@ -0,0 +1,475 @@
+(usage-getting-started)=
+
+# Getting started
+
+*Make sure you have followed the* [installation instructions](../Installation.md).
+
+In this tutorial you will learn how to:
+
+1. [Create a simple electrical network with one source and one load](gs-creating-network);
+2. [Solve a load flow](gs-solving-load-flow);
+3. [Get the results of the load flow](gs-getting-results);
+4. [Update the elements of the network](gs-updating-elements);
+5. [Save the network and the results to the disk for later analysis](gs-saving-network);
+6. [Load the saved network and the results from the disk](gs-loading-network).
+
+(gs-creating-network)=
+
+## Creating a network
+
+An electrical network can be built by assembling basic elements provided in [roseau.load_flow.models][api-models].
+The following is a summary of the available elements:
+
+* Buses:
```

Comment:
The link to `[api-models]` already exists after the short description of the elements. At this points, I think I would prefer to have a link to `/models`